.travis.yml

```diff
@@ -3,8 +3,7 @@ jdk:
   - openjdk8
 script:
   - mvn test -B
+  - bash .scripts/buildDB.sh
   - mvn clean package jetty:run &
-  # sleep to wait for jetty service to be ready
```

Comment:
This comment tells us why we need to sleep 30 secs. Why should it be deleted?

Reply:
Added.